src/coreclr/jit/fgdiagnostic.cpp

```diff
@@ -2573,6 +2573,10 @@ void Compiler::fgDumpBlockMemorySsaIn(BasicBlock* block)
         {
             printf(" = m:%u\n", block->bbMemorySsaNumIn[memoryKind]);
         }
+        else if (block->bbMemorySsaPhiFunc[memoryKind] == BasicBlock::EmptyMemoryPhiDef)
+        {
+            printf(" = phi([not filled])\n");
+        }
```

Comment:
I've hit crashes here a few times during JITDUMP when I've messed up the flow graph.